content/challenges/easy/2 - Life Stages/func.js

```diff
@@ -14,5 +14,16 @@
  * @returns {string}
  */
 module.exports = (age) => {
-    // Your code here
-}
\ No newline at end of file
+    if (age < 2) {
+        return "Baby";
+    } if (age >= 2 && age < 13) {
+        return "Kid";
+    } if (age >= 13 && age <= 19) {
+        return "Teen";
+    } if (age > 19 && age <= 30) {
+        return "Young Adult";
+    } else {
+        return "Dead";
+    }
```

Comment:
Nice! It shows me you understand how logic flows in code because you didn't add a condition like `(age > 30)` on that last else. It's tempting to do but if you understand logic and the discrete mathematics behind this logic you understand it's not necessary.

Overall this entire function looks spot on.